Felienne/hedy
# [BUG FIX] Login screen no longer suggests support of email login

**Description**
In this PR we remove the suggestion on the login page that users are able to login with the email address while this is not supported. As username and email where already two separate string in the yamls the fix only had to be made on the front-end. Resulting in a very very small PR.

**Fix for**
This PR fixes #1999.

**How to test**
Navigate to the login screen and notice the textual change from username / email to email.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
